src/modules/dashboard/components/ActionsPageFeed/ActionsPageEvent/ActionsPageEvent.tsx

```diff
@@ -427,39 +429,45 @@ const ActionsPageEvent = ({
             />
           </div>
           <div className={styles.details}>
-            <div className={styles.roles}>
-              {eventName &&
-                EVENT_ROLES_MAP[eventName] &&
-                EVENT_ROLES_MAP[eventName].map((role, index) => {
+            {eventName && EVENT_ROLES_MAP[eventName] && (
+              <div className={styles.roles}>
+                {EVENT_ROLES_MAP[eventName].map((role, index) => {
+                  /* If it is a smite action, do not display the Root label
+                    If it is an award action, do not display the Arbitration label */
                   if (
-                    (!values?.isSmiteAction && role === ColonyRole.Root) ||
-                    (values?.isSmiteAction && role !== ColonyRole.Root)
+                    (values?.actionType ===
+                      ColonyActions.EmitDomainReputationPenalty &&
+                      role === ColonyRole.Root) ||
+                    (values?.actionType ===
+                      ColonyActions.EmitDomainReputationReward &&
+                      role === ColonyRole.Arbitration)
```

Comment:
Is the `values.isSmiteAction` value still returned ?

If we're not using it anymore _(and it's not used elsewhere)_ please remove it

Reply:
It is being used at various other places, I took a look.